api/src/main/java/javax/enterprise/inject/spi/Annotated.java

```diff
@@ -51,18 +52,34 @@
      * 
      * @return a set of all types to which the base type should be considered assignable
      */
-    public Set<Type> getTypeClosure();
+    Set<Type> getTypeClosure();
 
     /**
      * <p>
      * Get program element annotation of a certain annotation type.
+     *
+     * Since 2.0, calling this method is considered deprecated.  To support {@see Repeatable} annotations, call {@see getAnnotation}
```

Comment:
Ok, I'll replace the `@Deprecated` with this as well.  Link was what I was looking for, but couldn't remember.